model/model.go

```diff
@@ -227,3 +227,22 @@ func (model Model) SortPoliciesByPriority() error {
 	}
 	return nil
 }
+
+func (model Model) ToText() string {
+	s := strings.Builder{}
+	s.WriteString("[request_definition]\n")
+	s.WriteString(fmt.Sprintf("r = %s\n", strings.Replace(model["r"]["r"].Value, "_", ".", -1)))
```

Comment:
Plz define `strings.Replace(string, "_", ".", -1)` in a seperated utility function